test/Dialect/Substrait/fetch.mlir

```diff
@@ -16,6 +16,21 @@ substrait.plan version 0 : 42 : 1 {
   }
 }
 
+// CHECK-LABEL: substrait.plan
+// CHECK:         relation
+// CHECK:           %[[V0:.*]] = named_table
+// CHECK-NEXT:      %[[V1:.*]] = fetch all offset 3 from %[[V0]]
+// CHECK-SAME:        : tuple<si32> 
+// CHECK-NEXT:      yield %[[V1]] : tuple<si32>
+
+substrait.plan version 0 : 42 : 1 {
+  relation {
+    %0 = named_table @t1 as ["a"] : tuple<si32>
+    %1 = fetch -1 offset 3 from %0 : tuple<si32> 
```

Comment:
We should at least *also*  test the parsing of `all`. Was that an oversight or did you want to check the correspondence to `-1`? If so, add another test case for `all`.
```suggestion
    %1 = fetch all offset 3 from %0 : tuple<si32> 
```

Reply:
Added an additional test case.